drivers/pwm/pwm_nrfx.c

```diff
@@ -126,8 +288,10 @@ static int pwm_nrfx_init(struct device *dev)
 static void pwm_nrfx_uninit(struct device *dev)
 {
 	const struct pwm_nrfx_config *pconfig = dev->config->config_info;
+	NRF_PWM_Type *pwm_instance = pconfig->pwm_regs;
```

Comment:
No such field in the `pwm_nrfx_config` structure.

Reply:
Thanks! Did not see this one as i have not tested this function. Will make it correct